spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewritePositionDeleteFilesAction.java

```diff
@@ -1054,48 +1054,48 @@ private void checkResult(
       List<DeleteFile> newDeletes,
       int expectedGroups) {
     assertThat(rewrittenDeletes.size())
-        .as("Expected rewritten delete file count does not match")
+        .as("Rewritten delete file count does not match")
```

Comment:
How about get rid of 'does not match, match', again in my thought, the assert message will already say this.

Reply:
Removed, then the failure logs will be like
```
java.lang.AssertionError: [XXX..: number of results should match] 
			  Actual and expected should have same size but actual size is:
```